src/main/java/swaglabs/common/CommonMethods.java

```
@@ -19,6 +19,8 @@
 

```

Method 'loginPageButton' contains empty catch block which suppresses exceptions. This is a poor practice that can hide errors.